commit 6450a10acaa927d38a04df1a29d97d2c9d5908de
Author: Zbigniew Halas <zbigniew.halas@citrix.com>
Date:   Tue Mar 13 15:59:58 2012 +0000

    Revert "Don't disable JavaScriptCore when building with mingw32"
    
    This reverts commit 3a7ae201047d2a5706689f2b3ccc318ec24450d1.

diff --git a/wscript b/wscript
index d6ef738..64d9f96 100644
--- a/wscript
+++ b/wscript
@@ -208,6 +208,7 @@ def configure (conf):
         args = '--define-variable=target=win32'
     check_pkg ('gtk+-2.0', '2.10.0', var='GTK', args=args)
     check_pkg ('webkit-1.0', '1.1.1', args=args)
+    conf.define ('HAVE_JSCORE', [0,1][not is_mingw (conf.env)])
     check_pkg ('libsoup-2.4', '2.25.2')
     conf.define ('HAVE_LIBSOUP_2_25_2', 1)
     check_pkg ('libxml-2.0', '2.6')
